README.md

```diff
@@ -111,6 +111,58 @@ Route::get('login_callback', function () {
 });
 ```
 
+## Mollie Recurring
```

Comment:
You could refer to some anchor links from our [Recurring documentation](https://mollie.dev/en/docs/recurring) if you like, to give the reader some additional information.